src/bosh_aws_cpi/lib/cloud/aws/resource_wait.rb

```diff
@@ -74,22 +74,16 @@ def self.for_volume(args)
 
     def self.for_volume_modification(args)
       volume_modification = args.fetch(:volume_modification) { raise ArgumentError, 'volume_modification object required' }
-      target_state = args.fetch(:state) { raise ArgumentError, 'state symbol required' }
+      target_states = args.fetch(:states) { raise ArgumentError, 'states array required' }
       valid_states = ['modifying', 'optimizing', 'completed', 'failed']
-      validate_states(valid_states, target_state)
-      wait_time_factor = args.fetch(:wait_time_factor, 1).to_i
+      target_states.each { |target_state| validate_states(valid_states, target_state) }
+      target_state_desc = target_states.join(' or ')
 
-      ignored_errors = []
```

Comment:
Why we don't need to ignore these errors any more? I couldn't follow this and doesn't seem to be related to this change.

Reply:
I bumped into this while reading my own copy/paste-based code. 😅

I figured that in the original code I've copied, we have to ignore "disk resource missing" errors while creating a new disk, because before it is actually created, some API calls may error in this way.

But when resizing a disk, then the API resource for the disk should definitely exist. So we don't ignore such errors.